lib/recurly/webhook/low_balance_gift_card_notification.rb

```diff
@@ -0,0 +1,6 @@
+module Recurly
+  module Webhook
+    class LowBalanceGiftCardNotification < SubscriptionNotification
+    end
+  end
+end
```

Comment:
Nit: you want your EOF to be on the last line. Files should end with newline + EOF.

Reply:
Good catch